R/content.R

```diff
@@ -747,6 +769,52 @@ terminate_jobs <- function(content, keys = NULL) {
   res_df
 }
 
+#' @rdname get_jobs
+get_job_list <- function(content) {
+  validate_R6_class(content, "Content")
+
+  purrr::map(content$jobs(), ~ purrr::list_modify(.x, client = content$connect))
+}
+
+#' Get Job Log
+#'
+#' Get the log output for a job. Requires Connect 2022.10.0 or newer.
+#'
+#' @param job A job, represented by an element from the list returned by `get_job_list()`.
+#' @param max_log_lines Optional. An integer indicating the maximum number of
+#' log lines to return. If `NULL` (default), Connect returns a maximum of 5000
+#' lines.
+#'
+#' @return A data frame with the requested log. Each row represents an entry.
+#'
+#' - `source`: `stdout` or `stderr`
+#' - `timestamp`: The time of the entry.
+#' - `data`: The logged text.
+#'
+#' @examples
+#' \dontrun{
+#' client <- connect()
+#' item <- content_item(client, "951bf3ad-82d0-4bca-bba8-9b27e35c49fa")
+#' jobs <- get_job_list(item)
+#' target_job <- jobs[[1]]
+#' log <- get_job_log(target_job)
```

Comment:
This definitely does read nicer to me. Maybe simplified with:

```
#' log <- get_job_log(jobs[[1]])
```

Reply:
I intentionally included that as a separate line to call it out a bit more clearly (by itself, with that variable name, that line says "this item is a job" rather than just "this item is an argument you pass to this function").

Reply:
But I made the change. Can I get a re-review?